packages/providers/src.ts/default-hedera-provider.ts

```diff
@@ -17,7 +16,6 @@ export enum HederaNetworks {
  */
 export class DefaultHederaProvider extends BaseProvider {
     consensusNodeUrl: string;
-
     constructor(network: Networkish, options?: {
```

Comment:
do we need the `options?` argument if we are providing that as part of the `network` argument and do not utilise it in any way in the `DefaultHederaProvider`